<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphics">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>671</height>
     </rect>
    </property>
    <property name="interactive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>461</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>НАЧАЛО ОТРЕЗКА</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>КООРДИНАТА X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="startX"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>КООРДИНАТА Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="startY"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>КОНЕЦ ОТРЕЗКА</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>КООРДИНАТА X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="endX"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>КООРДИНАТА Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="endY"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ВЫБЕРИТЕ АЛГОРИТМ ПОСТРОЕНИЯ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="algor">
       <item>
        <property name="text">
         <string>ЦДА</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Бреднхема с действительными числами</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Бреднхема с целыми числами</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Бреднхема с устранением ступенчатости</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Встроенный метод</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ВЫБЕРИТЕ ЦВЕТ ЛИНИИ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="colour">
       <item>
        <property name="text">
         <string>Черный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Красный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Оранжевый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Желтый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Зеленый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Голубой</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Синий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Фиолетовый</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="bgc">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Цвет фона</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="build">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ПОСТРОИТЬ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>370</y>
      <width>461</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ВВЕДИТЕ УГОЛ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="deg"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ВЫБЕРИТЕ АЛГОРИТМ ПОСТРОЕНИЯ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="algor_2">
       <item>
        <property name="text">
         <string>ЦДА</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Бреднхема с действительными числами</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Бреднхема с целыми числами</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Бреднхема с устранением ступенчатости</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Встроенный метод</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ВЫБЕРИТЕ ЦВЕТ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="colour_2">
       <item>
        <property name="text">
         <string>Черный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Красный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Оранжевый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Желтый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Зеленый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Голубой</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Синий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Фиолетовый</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="bgc_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Цвет фона</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="build_sp">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ПОСТРОИТЬ СПЕКТР</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>600</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ОЧИСТИТЬ ОБЛАСТЬ РИСОВАНИЯ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_all">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>640</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ОЧИСТИТЬ ВСЕ ПОЛЯ</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1088</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
